validate/schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "format": {
+      "type": "string",
+      "enum": [
+        "env",
+        "json",
+        "secret"
+      ]
+    },
+    "output": {
+      "type": "string"
+    },
+    "owner": {
+      "type": "integer"
+    },
+    "prefix": {
+      "type": "string"
+    },
+    "uppercase": {
+      "type": "boolean"
+    },
+    "secrets": {
+      "$ref": "#/$defs/secretsArray"
+    }
+  },
+  "required": [
+    "secrets"
+  ],
+  "$defs": {
+    "secretsArray": {
+      "type": "array",
+      "items": {
+        "anyOf": [
+          {
+            "type": "string",
+            "pattern": "^(?:\\S+\/)*\\S+$"
```

Comment:
I might have vacation brain - but can you explain this regex to me?

Reply:
![Screenshot 2022-08-03 at 11 31 15](https://user-images.githubusercontent.com/5990062/182575269-d5e6fbc0-bbd3-4233-9baa-8ce73886c10f.png)

So this just validates how the string/key looks inside the object.